bin/_mocha

```diff
@@ -295,9 +295,9 @@ if (program.retries) mocha.suite.retries(program.retries);
 
 var extensions = ['js'];
 program.compilers.forEach(function(c) {
-  var compiler = c.split(':')
-    , ext = compiler[0]
-    , mod = compiler[1];
+  var compiler = c.match(/([a-z.]+):(([A-Z]:)?\S*)+/)
```

Comment:
there are several problems with this regular expression--instead, try `c.split(':', 1)`.  does that work?